CONTRIBUTING.md

```diff
@@ -138,6 +138,19 @@ Because SQL within JSON or YAML will not generally be syntax highlighted, indent
 * One space before and after = etc
 * Name SQL subqueries after the layer name (but use underscores)
 
+## Map Icon Guidelines
+
+Design Guidelines
```

Comment:
This should be a level 3 heading, or more appropriately, combine the two lists into one.